### ISSUE TITLE
stats: v2.1.1 actix worker paniced with "index out of range"

### ISSUE THREAD

**Issue by Revolution1** [blockscout/blockscout-rs#1082]
**Describe the bug**

**To Reproduce**

1. Go to '/stats'
2. See error: some charts not shown, api returns 502, in the log it says:
    ```
    thread 'actix-server worker 1' panicked at 
    /rustc/129f3b9964af4d4a709d1383930ade12dfe7c081/library/alloc/src/vec/mod.rs:2197:36:
    range end index 1 out of range for slice of length 0
    ...
    thread 'actix-server worker 5' panicked at 
    /rustc/129f3b9964af4d4a709d1383930ade12dfe7c081/library/alloc/src/vec/mod.rs:2197:36:
    range end index 1 out of range for slice of length 0
    ```

**Screenshots**

![image](https://github.com/user-attachments/assets/ad8d4f29-982d-42ba-923e-54d30939e3a2)

**Environment**

- **image:** `ghcr.io/blockscout/stats:v2.1.1`
- **config.json:** the image built-in one

<details><summary>Environmental variables</summary>
<p>

```
DATABASE_URL : secret(avive-sepolia-blockscout-v2-env)[DATABASE_URL] 
SECRET_KEY_BASE : secret(avive-sepolia-blockscout-v2-env)[SECRET_KEY_BASE] 
STATS_CHARTS__TEMPLATE_VALUES__NATIVE_COIN_SYMBOL : Avive
STATS__BLOCKSCOUT_DB_URL : secret(avive-sepolia-blockscout-v2-env)[STATS__BLOCKSCOUT_DB_URL] 
STATS__CREATE_DATABASE : true
STATS__DB_URL : secret(avive-sepolia-blockscout-v2-env)[STATS__DB_URL] 
STATS__METRICS__ADDR : 0.0.0.0:6060
STATS__METRICS__ENABLED : true
STATS__METRICS__ROUTE : /metrics
STATS__RUN_MIGRATIONS : true
STATS__SERVER__GRPC__ADDR : 0.0.0.0:8051
STATS__SERVER__GRPC__ENABLED : false
STATS__SERVER__HTTP__ADDR : 0.0.0.0:8050
STATS__SERVER__HTTP__ENABLED : true
STATS__SERVER__HTTP__MAX_BODY_SIZE : 2097152
```

</p>
</details>

**Comment by bragov4ik** [comment]
Hi, thanks for the provided details! I can't reproduce it with provided  Could you please also try to run it with `RUST_BACKTRACE=1` env and show the stacktrace on panic? It's very hard to understand if it's the problem with stats service or somewhere down the line :(

Also, at the same time it would be great to also add `RUST_LOG=debug` and show nearby logs :) (or even `RUST_LOG=trace` but it could be too wordy).

**Comment by bragov4ik** [comment]
Btw, I see that https://sepolia.avive.world/stats works fine and it's at least `v2.0.0`. Does the issue appear when upgrading from `v2.0.0` to `v2.1.1`?

**Comment by bragov4ik** [comment]
I also ran some fuzz tests locally with randomly generated vecs as input and did not manage to encounter the panic; so the info I mentioned would be great as a starting point for investigation :)

**Comment by bragov4ik** [comment]
feel free to reopen if the issue persists
